test/e2e/tests/tokens/token-list.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('Token List', function () {
           tokenAddress,
           '+0.05%',
         );
-        await assetListPage.check_tokenGeneralChangeValue('+$50.00');
+        await assetListPage.check_tokenGeneralChangeValue('+$50');
```

Comment:
so I think we need a condition like this:

`if (process.env.SELENIUM_BROWSER === Browser.FIREFOX)` --> `+$50`

because on Chrome we do expect to see `+$50.00`